scripts/ImgChecker/ImgChecker.rb

```diff
@@ -0,0 +1,80 @@
+# ImgChecker, checks for big images and reports back to Travis making the
+# build fail. Made with love for FOSSASIA.
+
+# Dependencies
+require 'fastimage'
+
+# Program start message
+puts "Starting ImgChecker, authored by Abishek V Ashok for FOSSASIA with love"
+
+# Initializing the directories into variables
+studentsImgFiles = Dir["./img/students/**/*.*"]
+gImgFiles = Dir["./img/students/**/*.*"]
+mentorsImgFiles = Dir["./img/mentors/**/*.*"]
+blogsImgFiles = Dir["./img/blogs/**/*.*"]
+privlyImgFiles = Dir["./img/privly/**/*.*"]
+
+# Defines the standard maximum dimensions for images in each section
+studentsImgSize = 240
+gImgSize = 240
+mentorsImgSize = 240
+blogsImgSize = 240
```

Comment:
@robbyoconnor All images for Blog Posts are 240x240 and that's why [tests](https://travis-ci.org/fossasia/gci15.fossasia.org/builds/184755491) are passing. :-)